src/devices/efergy_e2_classic.c

```diff
@@ -60,24 +68,59 @@ static int efergy_e2_classic_callback(bitbuffer_t *bitbuffer) {
 		return 0;
 	}
 
-	const unsigned VOLTAGES[] = {110, 115, 120, 220, 230, 240, 0};
-
-	double current_adc = 256 * bytes[4] + bytes[5];
-	for (unsigned i = 0; VOLTAGES[i] != 0; ++i) {
-		double power  = (VOLTAGES[i] * current_adc * (1 << bytes[6])) / 32768;
-		fprintf(stderr, "Power consumption at %u volts: %.2f watts\n", VOLTAGES[i], power);
+	if (bytes[6] > 0x80)
```

Comment:
shouldn't this be `>=` (or `&0x80`) as 0x80 would already be a negative value? But I'm not entirely following, isn't bytes[6] 2s-complement? Wouldn't a cast to `signed char`do?